etl/steps/data/garden/cait/2022-08-10/ghg_emissions_by_sector.py

```diff
@@ -0,0 +1,229 @@
+from typing import Dict, List
+
+import pandas as pd
+from owid import catalog
+from owid.datautils import dataframes, geo
+
+from etl.paths import DATA_DIR
+from shared import CURRENT_DIR
+
+# Details for dataset to export.
+DATASET_SHORT_NAME = "ghg_emissions_by_sector"
```

Comment:
(I'm not suggesting that you use this, it's just a syntactic sugar. So totally up to you. You'd have to rebase to use it)

You could give `Names` helper object a try and replace these constants by
```
from etl.helpers import Names
N = Names(__file__)
```
and then use `N.country_mapping_path` in code instead of `COUNTRY_MAPPING_PATH `

Reply:
Thanks, I think I'll start using Names for the next one.